### ISSUE TITLE
DataFrame multi-index doesn't support column names

### ISSUE THREAD

**Issue by snorfalorpagus** [pywr/pywr#263]
The new feature introduced in #262 requires that data loaded from excel have it's column specified by index (0, 1, ...) rather than by name. Is this an issue for upstream?

**Comment by jetuk** [comment]
I believe it works OK if `url` is a CSV file.

I also think one of the reasons is that it supports a list of column letters (e.g. `['A', 'B']`).

**Comment by snorfalorpagus** [comment]
We could implement this by calling `.set_index()` on the data frame once it has been loaded.

Alternatively we can leave it as-is and just document it.

**Comment by jetuk** [comment]
Yes I guess it depends if the docs say "other" keywords as passed to the respective `pandas.read_` method.
